### PR TITLE
Add meeting 30 blog post

### PR DESCRIPTION
**Pull request by adelpopelkova**

### CODE REVIEW

File: freezeyt_blog/articles/meeting30.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+> které odpovídají danému regulárnímu výrazu.
+> regulární výraz `[^-a-zA-Z0-9._/:]` odpovídá všemu bez pomlčky, tečky,
+> podtržítka, lomítka, dvojtečky velkých a malých písmen (bez diakritiky)
+> a číslic v názvu souboru.
+
+Na chvíli jsme se zastavili u příkazu `git cherry-pick` - ten může na danou
+větev přidat i commit, ke kterému se nedostaneme z žádné větve
+(protože byla například smazána).
+Dostaneme se k němu právě proto, že známe jeho commit hash.
+V Gitu existuje [příkaz](), který by tyto commity ze „vzduchoprázdna“ dokáže vylovit.
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting30.md:76]: I am surprised, that GH Actions didn't have a problem with the incomplete link, last time I tried to do this in a freezeyt blog article it ended with an error saying it found a broken link.

File: freezeyt_blog/articles/meeting30.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Freezeyt sraz dvacátý osmý - mazání adresářů
+
+Na začátku jsme se koukli na Pull requasty od minula - první přidával článek
```
Comment (encukou) [inline freezeyt_blog/articles/meeting30.md:3]:
```suggestion
Na začátku jsme se koukli na Pull requesty od minula - první přidával článek
```

File: freezeyt_blog/articles/meeting30.md (hunk 1 above)
Comment (adelpopelkova) [reply freezeyt_blog/articles/meeting30.md:76]:
```suggestion
V Gitu existuje [příkaz](https://git-scm.com/docs/git-fsck#Documentation/git-fsck.txt---no-reflogs), který by tyto commity ze „vzduchoprázdna“ dokáže vylovit.
```

Comment (encukou) [reply freezeyt_blog/articles/meeting30.md:76]: The command is `git fsck`.

**Review by encukou — APPROVED** [review]

**Pull request merged**